Firmware/Marlin_main.cpp

```diff
@@ -2058,6 +2106,8 @@ void process_commands()
   float tmp_motor_loud[3] = DEFAULT_PWM_MOTOR_CURRENT_LOUD;
   int8_t SilentMode;
 #endif
+  KEEPALIVE_STATE(IN_HANDLER);
```

Comment:
No, with this change it won't send anything anymore. Is this PR based on the recent Marlin Codebase or some early test branch?